backend/spec/requests/surveys_request_spec.rb

```diff
@@ -18,6 +18,7 @@
         'name' => survey.name,
         'description' => survey.description,
         'survey_subject_id' => survey_subject.id,
+        'answered_questions_quantity' => 1,
```

Comment:
Same as the comment above.
```suggestion
        'answered_questions_quantity' => survey.answered_questions_quantity,
```

Reply:
At a conference, they said it is better to just hard code the result in the specs as if you make a mistake, it will show the mistake in the specs as well.